### DISCUSSION TITLE
VIcon append-icon prepend-icon offline mode

### DISCUSSION THREAD

**Discussion by karis** [gitsindonesia/ui-component#125] [Q&A] [answered]
hi @gravitano ,

i have a customer, and they cant online their server. they have to go to the nearest city. so they can get online.
as we know ui-component comes with online iconify library.
so how can we set ui-component to use offline bundle icon?

**Comment by gravitano** [comment, marked as answer]
Not tested but I think you can follow this docs to support offline docs https://docs.iconify.design/icon-components/vue/offline.html

**Reply by karis** [reply]
ill try it first, or second strategy i will store svg as vue components.

thank you @gravitano
